sgpc3/Makefile

```diff
@@ -3,7 +3,8 @@ sgp_common_dir := ../sgp-common
 sw_i2c_dir := ${sensirion_common_dir}/sw_i2c
 hw_i2c_dir := ${sensirion_common_dir}/hw_i2c
 
-CFLAGS := -Wall -Wstrict-aliasing=1 -I. -I${sensirion_common_dir} -I${sgp_common_dir}
+CFLAGS := -Wall -fstrict-aliasing -Wstrict-aliasing=1 \
```

Comment:
Why don't we have the -Os flag for sgpc3?

Reply:
someone slacked :face_with_head_bandage: